plugins/optimization-detective/hooks.php

```diff
@@ -15,3 +15,5 @@
 OD_URL_Metrics_Post_Type::add_hooks();
 add_action( 'wp', 'od_maybe_add_template_output_buffer_filter' );
 add_action( 'wp_head', 'od_render_generator_meta_tag' );
+add_filter( 'site_status_tests', 'od_optimization_detective_add_rest_api_test' );
+add_action( 'admin_init', 'od_rest_api_health_check_plugin_activation' );
```

Comment:
Since this is for rendering admin notices, shouldn't it use the `admin_notices` action instead?

Reply:
Oh, good catch! This is getting output _before_ the DOCTYPE at present!

![image](https://github.com/user-attachments/assets/4fe6624c-9790-46c7-85df-1fb602549977)

The lack of the `inline` class is resulting in the admin JS moving the notice to the right spot.

Reply:
Fixed in 7ad36cd94b72a3ad27a2714cd93575f19023cdb6